inlong-manager/manager-workflow/src/main/java/org/apache/inlong/manager/workflow/core/impl/TaskServiceImpl.java

```diff
@@ -67,6 +68,8 @@ public WorkflowContext transfer(Integer taskId, String remark, List<String> to,
     public WorkflowContext complete(Integer taskId, String remark, String operator) {
         WorkflowContext context = workflowContextBuilder
                 .buildContextForTask(taskId, WorkflowAction.COMPLETE, remark, operator);
+        ServiceTask serviceTask = (ServiceTask) context.getActionContext().getTask();
```

Comment:
Are the results of `context.getActionContext().getTask()` all `ServiceTask`?

Reply:
no，just one